diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0c72e86..e094b8d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,6 +36,10 @@ include(CompilerFlags)
 # Registers build options that are exposed to cmake
 include(CMakeOptions.txt)
 
+if(CMAKE_SYSTEM_NAME STREQUAL "iOS")
+  enable_language(OBJC)
+endif()
+
 if (UNIX AND NOT APPLE)
   if (NOT GLFW_BUILD_WAYLAND AND NOT GLFW_BUILD_X11)
     MESSAGE(FATAL_ERROR "Cannot disable both Wayland and X11")
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 47d2c1e..14a8be2 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -54,6 +54,10 @@ else ()
     MESSAGE(STATUS "Audio Backend: None (-DCUSTOMIZE_BUILD=ON -DSUPPORT_MODULE_RAUDIO=OFF)")
 endif ()
 
+if(CMAKE_SYSTEM_NAME STREQUAL "iOS")
+    set_source_files_properties(raudio.c PROPERTIES LANGUAGE OBJC)
+endif()
+
 add_library(raylib ${raylib_sources} ${raylib_public_headers})
 
 if (NOT BUILD_SHARED_LIBS)
diff --git a/cmake/GlfwImport.cmake b/cmake/GlfwImport.cmake
index 92cd5c3..fc0e0a7 100644
--- a/cmake/GlfwImport.cmake
+++ b/cmake/GlfwImport.cmake
@@ -32,6 +32,8 @@ elseif("${PLATFORM}" STREQUAL "DRM")
     MESSAGE(STATUS "No GLFW required on PLATFORM_DRM")
 elseif("${PLATFORM}" STREQUAL "Android")
     list(REMOVE_ITEM LIBS_PRIVATE glfw)
+elseif("${PLATFORM}" STREQUAL "SDL")
+    MESSAGE(STATUS "No GLFW required on PLATFORM_SDL")
 else()
     MESSAGE(STATUS "Using external GLFW")
     set(GLFW_PKG_DEPS glfw3)
